pkg/network/ebpf/c/protocols/tls/https.h

```diff
@@ -29,33 +29,19 @@
 
 #define HTTPS_PORT 443
 
-static __always_inline int http_process(http_transaction_t *http_stack, skb_info_t *skb_info, __u64 tags);
+static __always_inline bool http_process(http_transaction_t *http_stack, skb_info_t *skb_info, __u64 tags);
 
 static __always_inline void https_process(conn_tuple_t *t, void *buffer, size_t len, __u64 tags) {
```

Comment:
why did you make that change here?

Reply:
it will probably make more sense if you take a look at the commit as a whole:
https://github.com/DataDog/datadog-agent/pull/16167/commits/a2d83af25904919fec34ddd8675d35e9c3c05462

specifically at the change in `http_process` and `https_process`

Reply:
thanks!
my concern is that we are now not able to classify the underlying protocol (unless it's http) while we could classify it

Reply:
Hmmm, true 😓 
I changed the code to use `classify_protocol_for_dispatcher`